packages/ecc-utils-design/src/components/form/form.ts

```diff
@@ -246,6 +257,34 @@ export default class EccUtilsDesignForm extends LitElement {
     `;
   }
 
+  private renderCodeTemplate(field: Field, path: string): TemplateResult {
+    if (field.type !== "code") return html``;
+    // TODO: add CSS parts
+
+    const { fieldOptions, codeOptions } = field;
+    const { formControl, formControlLabel } = this.cssParts;
+
+    return html`<ecc-utils-design-code
+      part="form-control: ${formControl},
```

Comment:
by setting the part like this you're setting these parts to control the entire component and that messing with its styles. You only need to export the already created parts with `exportparts`.

Reply:
please check if this is what you meant.